app/src/main/resources/application.properties

```diff
@@ -3,11 +3,11 @@
 
 # System Info
 
-app.name=Apicurio API Designer
-app.description=The spiritual successor to Apicurio Studio - used as the basis for Red Hat OpenShift API Designer.
-app.version=${project.version}
-app.date=${timestamp}
-app.git.commit-id=${git.commit.id.full}
+apicurio.name=Apicurio API Designer
+apicurio.description=The spiritual successor to Apicurio Studio - used as the basis for Red Hat OpenShift API Designer.
+apicurio.version=${project.version}
+apicurio.date=${timestamp}
+apicurio.git.commit-id=${git.commit.id.full}
```

Comment:
While I don't like "app" as an initial prefix, I'm wondering if the general format of our properties should be something like this:

`apicurio.CATEGORY.property.hierarchical.name`

For `CATEGORY` we would have things like `auth` and `ui` and `log` etc.  The category would allow us to organize similar properties in documentation and in the UI (for dynamic properties).

In that case, we should consider a category for these properties.  Perhaps `apicurio.app.` for these?  Or maybe `apicurio.app-info.`?  Or maybe just `apicurio.info.`?

Thoughts?

Reply:
Or `apicurio.system`/`apicurio.system-info` to align it with https://github.com/Apicurio/apicurio-api-designer/blob/main/app/src/main/resources-unfiltered/META-INF/apis/designer/v0/designer-api-v0.json#L18

Reply:
Yes, I think this makes sense. I added apicurio.app as the prefix.